src/main/app/api/to-api-format.ts

```diff
@@ -20,9 +20,6 @@ const fields = {
 };
 
 const toApiDate = (date: CaseDate) => {
-  if (!date?.year || !date?.month || !date?.day) {
```

Comment:
Removing this might cause one of the functional tests to fail, as it will try to submit or save and sign out with a partially complete day/month/year field.

Perhaps we need to complete the "API failed to save" story to handle this situation.